modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/transformer/hdbdd/HdbddTransformer.java

```diff
@@ -383,11 +376,24 @@ private List<EntityElementSymbol> getStructuredTypeSubElements(FieldSymbol entit
     return subElements;
   }
 
-  private XSKDataStructureHDBTableColumnModel getAssociationForeignKeyColumn(AssociationSymbol associationSymbol, EntityElementSymbol foreignKey){
+  private XSKDataStructureHDBTableColumnModel getAssociationForeignKeyColumn(AssociationSymbol associationSymbol,
+      EntityElementSymbol foreignKey) {
     XSKDataStructureHDBTableColumnModel columnModel = transformFieldSymbolToColumnModel(foreignKey, false);
     columnModel.setPrimaryKey(associationSymbol.isKey());
     columnModel.setNullable(!associationSymbol.isNotNull());
 
     return columnModel;
   }
+
+  private String getFullTableName(ViewSymbol dependingView, String tableName) {
+    if (tableName.equalsIgnoreCase(DUMMY_TABLE)) {
```

Comment:
What is this `DUMMY_TABLE`?

Reply:
There was a comment that explains this, I've added it again.